### ISSUE TITLE
Assuming where wp-config.php is located causing failure and PHP warnings

### ISSUE THREAD

**Issue by knutsp** [terrylinooo/cache-master#5]
```
PHP Warning:  fopen(/home/*/public_html/wp-config.php): failed to open stream: No such file or directory in /home/*/public_html/wp-content/plugins/cache-master/inc/admin/functions.php on line 219
PHP Warning:  fgets() expects parameter 1 to be resource, bool given in /home/*/public_html/wp-content/plugins/cache-master/inc/admin/functions.php on line 225
PHP Warning:  fclose() expects parameter 1 to be resource, bool given in /home/*/public_html/wp-content/plugins/cache-master/inc/admin/functions.php on line 233
```
`inc/admin/functions.php` line 217:
`$wp_config_file = ABSPATH . 'wp-config.php';`

Problem occurs when wp-config.php is in parent folder of ABSPATH.

**Comment by terrylinooo** [comment]
Thanks for reporting. It will be fixed in next update.

**Comment by terrylinooo** [comment]
This issue has been fixed in current version (2.1.0)
